packages/emotion/src/emotion.ts

```diff
@@ -1,18 +1,38 @@
 import createEmotion from 'create-emotion';
 
+// We're doing this here because doing it globally was proving problematic.
+// We should solve for this if we need to use __TARGET__ elsewhere in the future.
+declare const __TARGET__:
```

Comment:
Updated the comment.

Also, that makes sense. I was originally just making a comprehensive list since this code in theory _technically_ supports all possible targets, but I updated this because it might prove confusing.